src/server/server_family.cc

```diff
@@ -1930,8 +1930,26 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
   }
 
   if (should_enter("PERSISTENCE", true)) {
-    auto save_info = GetLastSaveInfo();
+    size_t current_snap_keys = 0;
+    size_t total_snap_keys = 0;
+    double perc = 0;
+    {
+      lock_guard lk{save_mu_};
+      if (save_controller_) {
+        auto res = save_controller_->GetCurrentSnapshotProgress();
```

Comment:
you can move this to line 1967 right? we already have this code
      lock_guard lk{save_mu_};
      if (save_controller_) {
there